src/proj.h

```diff
@@ -1504,6 +1504,688 @@ double PROJ_DLL proj_coordinate_metadata_get_epoch(PJ_CONTEXT *ctx,
 
 /**@}*/
 
+/* ------------------------------------------------------------------------- */
+/* Binding in C of advanced methods from the C++ API                         */
+/*                                                                           */
+/* Manual construction of CRS objects.                                       */
+/* ------------------------------------------------------------------------- */
+
+/**
+ * \defgroup advanced_cpp_binding Binding in C of advanced methods from the C++
```

Comment:
```suggestion
 * \defgroup advanced_cpp_types Binding in C of advanced methods from the C++
```
and perhaps a slightly different title too.

Then start another group `advanced_cpp_functions` around L1592